### PR TITLE
Updated code owners, fixed some spelling errors and improved sections of README

### PR DESCRIPTION
**Pull request by vandyliu**
# Description:

Fixed some spelling errors and improved sections of README.
- Changed how images are displayed in README because one of the gifs wasn't loading properly before
- Updated code owners to reflect new garage interns
- Removed/changed instances of the name "Pacifica"
- Updated how links are displayed in README for readability
- Other grammar/spelling errors

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

# Limitations:

Please describe limitations of this PR

# Testing:

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- Manual tests done to make sure changing the name "Pacifica" did not break anything

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/constants.ts
Comment (nasadigital) [inline src/constants.ts:21]: I'm not super familiar with this part. Are you familiar with whether this needs to be changed in [package.json](https://github.com/microsoft/vscode-python-devicesimulator/blob/users/t-vali/spelling-errors/package.json#L177) as well?

File: docs/how-to-use.md
Comment (nasadigital) [inline docs/how-to-use.md]: I'm not a native speaker so I was wondering how does "opened **in** the simulator" sound compared to "opened **on** the simulator"?

File: README.md
Comment (nasadigital) [inline README.md]: Are quotation marks missing before "Device Simulator Express: New File"?

File: README.md
Comment (nasadigital) [inline README.md]: It seems that the command ("Device Simulator Express: New File") is formatted as inline code here, but a few lines above it's formatted as normal text.
I think it is easier for users to follow if we are consistent with the command formatting throughout the README (up to you to decide which formatting makes more sense).

**Review by nasadigital — COMMENTED** [review]
Great job, overall! 👍

File: README.md
Comment (vandyliu) [reply README.md]: Good catch, fixed in next commit

File: README.md
Comment (vandyliu) [reply README.md]: Very true, updated in next commit

File: src/constants.ts
Comment (vandyliu) [reply src/constants.ts:21]: Good catch, updated in next commit

File: docs/how-to-use.md
Comment (vandyliu) [reply docs/how-to-use.md]: I agree with you that it sounds better. Will update. :)

File: README.md
Comment (nasadigital) [inline README.md]: _&ast;nit&ast;_
Maybe use the same character for opening and closing quotation marks?

**Review by nasadigital — APPROVED** [review]
Thanks for applying the fixes!
LGTM :)

File: README.md
Comment (vandyliu) [reply README.md]: Completely agree. Updated.

**Review by andreamah — APPROVED** [review]
All changes seem good! Thanks for fixing the docs :)

**Comment by nasadigital** [comment]
Spotted one more typo. If you go to line 131 on [README.md](https://github.com/microsoft/vscode-python-devicesimulator/blob/users/t-vali/spelling-errors/README.md), I think it should be "open**s**" instead of "open" in "Opens an unsaved .py file with template code, also open the simulator.".

File: docs/install.md
Comment (isadorasophia) [inline docs/install.md:23]: (That's not on the scope of this PR, so don't worry!) I just realized we don't have instructions for building and installing the extension locally, was there any reason for that?

File: README.md
Comment (isadorasophia) [inline README.md]: Maybe we can point out the reason for that? E.g. [this reference](https://learn.adafruit.com/welcome-to-circuitpython/creating-and-editing-code#naming-your-program-file-7-32) or explain that this is the convention for CircuitPython.

**Review by isadorasophia — APPROVED** [review]
Awesome! ✨

File: README.md
Comment (vandyliu) [reply README.md]: Will update this in next commit

File: docs/install.md
Comment (vandyliu) [reply docs/install.md:23]: We do, but I think it was hard to find/understand because of the wording so I updated it in the next commit. Let me know what you think. I wasn't sure if I should add a whole "Contributing" section or if just adding the doc to set up in the "Useful Links" section is good enough.

**Review by xnkevinnguyen — APPROVED** [review]
Great PR!

File: .github/CODEOWNERS
Comment (markAtMicrosoft) [inline .github/CODEOWNERS]: After some consultation, Stephane and I agree, at least for now, please remove interns from CODEOWNERS file.  Coaches should remain.  Also remove @LukeSlev.  Because this is a live code base, we'd like another set of eyes.  The rules are basically that 2 approves are required and one of them must be code owner

**Review by markAtMicrosoft — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS
Comment (xnkevinnguyen) [reply .github/CODEOWNERS]: Ok, we will update accordingly, Thanks

Comment (vandyliu) [reply .github/CODEOWNERS]: Done!

**Review by markAtMicrosoft — APPROVED** [review]

File: .github/CODEOWNERS
Comment (LukeSlev) [reply .github/CODEOWNERS]: byebye

**Pull request merged**
